R/bin_time.R

```diff
@@ -114,8 +123,8 @@ bin_time <- function(occdf, bins, method = "mid", reps = 100,
     if (is.data.frame(bins) == FALSE) {
       stop("`bins` should be a dataframe.")
     }
-    if (any(is.na(occdf$max_ma)) || any(is.na(occdf$min_ma))) {
-      stop("NA values detected in occdf$max_ma or occdf$min_ma.")
+    if (any(is.na(occdf[, max_ma])) || any(is.na(occdf[, min_ma]))) {
+      stop(paste("NA values detected in", max_ma, "or", min_ma))
     }
```

Comment:
Perhaps outside the scope of this PR, but I wonder if it would be more user friendly to have an `na.action` argument that would specify how to proceed in these cases? For example, "na.omit" would just remove these rows and continue, whereas "na.fail" would raise this error.